### ISSUE TITLE
Accessibility: WCAG2AA.Principle3.Guideline3_1.3_1_1.H57.2

### ISSUE THREAD

**Issue by funkysi1701** [funkysi1701/funkysi1701.github.io#2861]
## Accessibility Issue Found by Pa11y

**WCAG Code**: `WCAG2AA.Principle3.Guideline3_1.3_1_1.H57.2`
**Total Occurrences**: 1
**URL**: https://blog-dev.funkysi1701.com
**Build**: 10.1.1.2484

---

### Example 1

**Message**: The html element should have a lang or xml:lang attribute which describes the language of the document.
**Selector**: `html`

**Context:**
```html
<html><head><title>503 Service Tempor...</html>
```

---
_Detected by [Pa11y](https://pa11y.org/) accessibility checker in Azure Pipeline_
